pandas/core/window/rolling.py

```diff
@@ -411,18 +393,39 @@ def _insert_on_column(self, result: "DataFrame", obj: "DataFrame"):
                 # insert at the end
                 result[name] = extra_col
 
-    def _center_window(self, result: np.ndarray, window) -> np.ndarray:
+    def calculate_center_offset(self, window, center) -> int:
```

Comment:
type the args

Reply:
I left `window` untyped since it can be either an ndarray or an int, but I typed `center`